recipes/librhash/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+import os
+from conans import CMake, ConanFile, tools
+
+
+class LibRHashConan(ConanFile):
+    name = "librhash"
+    description = "Great utility for computing hash sums"
+    topics = ("conan", "rhash", "hash", "checksum")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://rhash.sourceforge.net/"
+    license = "MIT"
+    exports_sources = "CMakeLists.txt",
+    generators = "cmake",
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False], "with_openssl": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_openssl": False,}
+
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+        if self.settings.os == "Windows" or self.options.shared:
```

Comment:
prefer config_options when removing fPIC, keeping a standard is better for reviewing

Reply:
Ok, `fPIC` is now removed in both `config_options` if on Windows and in `configure` if shared.